mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -242,7 +296,7 @@ DistanceSensorItem::Ptr DistanceSensorItem::create_item(DistanceSensorPlugin *ow
 		}
 
 		// optional
-		pnh.param("covariance", p->covariance, 0);
+		if (pnh.getParam("covariance", p->covariance) > 0) p->cov_is_def = true;
```

Comment:
Probably I can put it this way:
`pnh.param("covariance", p->covariance, -1);`
and then in line 234:
`if (covariance >= 0) covariance_ = covariance;`
Right?